app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -37,20 +37,40 @@ export class SearchAndAddOrder extends React.Component {
           <Tab
             tabName="Outpatient">
             <SearchDrug />
+            {
+              this.props.drug.uuid ?
+                <AddForm
+                  drugName={this.props.drug.display}
+                  drugUuid={this.props.drug.uuid}
+                />
+                : <div />
+            }
           </Tab>
           <Tab tabName="Inpatient">
             <SearchDrug />
+            {
+              this.props.drug.uuid ?
+                <AddForm
+                  drugName={this.props.drug.display}
+                  drugUuid={this.props.drug.uuid}
+                />
+                : <div />
+            }
           </Tab>
         </Tabs>
       </div>
     );
   }
 }
-
-const mapStateToProps = ({ careSettingReducer }) => ({
-  inpatientCareSetting: careSettingReducer.inpatientCareSetting,
-  outpatientCareSetting: careSettingReducer.outpatientCareSetting,
-});
+const mapStateToProps = (state) => {
+  const drug = state.drugSearchReducer.selected;
+  const { inpatientCareSetting, outpatientCareSetting } = state.careSettingReducer;
```

Comment:
You may consider destructing drugSearchReducer and careSettingReducer directly from the state